### ISSUE TITLE
FlyoutShowMode.TransientWithDismissOnPointerMoveAway not working

### ISSUE THREAD

**Issue by zhuxb711** [microsoft/microsoft-ui-xaml#8896]
### Describe the bug

Set Flyout.ShowMode = FlyoutShowMode.TransientWithDismissOnPointerMoveAway would not close the flyout even the mouse moves away

### Steps to reproduce the bug

```csharp
PointerPoint Point = <Your Mouse Point>;

Your_Flyout.ShowAt(Your_Element, new FlyoutShowOptions
{
    Position = new Point(Point.Position.X, Point.Position.Y),
    ShowMode = FlyoutShowMode.TransientWithDismissOnPointerMoveAway,
    Placement = FlyoutPlacementMode.BottomEdgeAlignedLeft,
})
```

### Expected behavior

FlyoutShowMode.TransientWithDismissOnPointerMoveAway works in WinUI2 but failed in WinUI3.

### Screenshots

![GIF 9-21-2023 11-25-08 AM](https://github.com/microsoft/WindowsAppSDK/assets/42038175/87ab8276-6511-4368-9331-becb6b6e84af)

### NuGet package version

Windows App SDK 1.4.0: 1.4.230822000

### Packaging type

Packaged (MSIX)

### Windows version

Windows 11 version 22H2 (22621, 2022 Update)

### IDE

Visual Studio 2022

**Comment by BendicoE** [comment]
Any news on this? I can confirm that this issue is still present in Windows App SDK v 1.4.231219. I might add the following: The dismissal on pointer move away seems to work the first time the flyout is shown (programmatically). On subsequent shows of the flyout, one has to tap outside to dismiss it.

**Comment by zhuxb711** [comment]
@BendicoE Still exists in Windows App SDK v1.5.0
@bpulliam @duncanmacmichael Could please share any news about this bug? 6 months pass seems you guys even not take a look at this issue.

**Comment by codendone** [comment]
@BendicoE Thanks for that comment about this only working the first time! The problem was the flyout code forgot to clear variable on close. This resulted in the flyout thinking when it was next shown that it didn't need to re-register an event listener to watch for pointer movement.

The fix is now in for 1.6.

**Comment by zhuxb711** [comment]
@codendone I think it's a simple fix and should be included in the next 1.5.X rather than wait for a long time until 1.6
